### ISSUE TITLE
Option for file input to wait for output to process request

### ISSUE THREAD

**Issue by buger** [buger/goreplay#426]
Sometimes you record traffic high-throughput environment processing lot of requests, and target you are replying too can't handle such load. Current way to solve the problem is to limit input by absolute value per second or by percentage, for example, replay on 1% of original speed: `--input-file "./requests.gor|1%"`. 

It would be nice to have an option to wait for output confirmation. For example, if we directly set `--output-http-workers` count, it will guarantee that no HTTP timeouts will happen.

**Comment by nagendra547** [comment]
Yes, this is very much needed.  So far I am able to make it work with "./requests.gor|1%", but yea putting "wait for output confirmation" will be much better solution.

**Comment by buger** [comment]
@nagendra547 my initial suggesting with setting `--output-http-workers 1` actually may work.
